samples/I2C.js

```diff
@@ -121,27 +120,29 @@ function changeRGB(red, green, blue) {
 }
 
 function writeWord() {
-    resetCursor();
-
+    clear();
```

Comment:
I'm hoping you don't need to clear the whole screen here? Can you move resetCursor() from below back to here and not have to change this line?

I just expect it will be slow and flicker more, i.e. bad practice for an example.